mycore-restapi/src/main/java/org/mycore/restapi/v1/MCRRestAPIClassifications.java

```diff
@@ -130,18 +130,7 @@ private static void writeChildrenAsJSON(Element eParent, JsonWriter writer, Stri
             writer.beginObject();
             writer.name("ID").value(e.getAttributeValue("ID"));
             writer.name("labels").beginArray();
-            for (Element eLabel : e.getChildren("label")) {
-                if (lang == null || lang.equals(eLabel.getAttributeValue("lang", Namespace.XML_NAMESPACE))) {
-                    writer.beginObject();
-                    writer.name("lang").value(eLabel.getAttributeValue("lang", Namespace.XML_NAMESPACE));
-                    writer.name("text").value(eLabel.getAttributeValue("text"));
-                    if (eLabel.getAttributeValue("description") != null) {
-                        writer.name("description").value(eLabel.getAttributeValue("description"));
-                    }
-                    writer.endObject();
-                }
-            }
-            writer.endArray();
+            extracted(writer, lang, e);
```

Comment:
Pleas provider a proper method name

Reply:
This was properly named in my latest commit.